bundles/org.openhab.binding.ism8/README.md

```diff
@@ -49,28 +53,34 @@ Description:
+
+Depending on the firmware version of the ISM8 and the connected systems, the supported IDs differ.
+- type="1.001"    - Please enter here the KNX type of the data point.
 You can find the data type in the Wolf ISM8 document as well.
 - write=true      - This parameter defines if the channel is bidirectional, but the parameter is optional and by default false.
 
-Note:
-Not all available types of the ISM8 interface are fully supported, but this can be extended.
-For the moment the following data types are implemented:
-
-| Channel type   | Datapoint type                 | Item type                 | R/W | KNX-type's                 |
-|----------------|--------------------------------|---------------------------|-----|----------------------------|
-| switch-rw      | Digital DataPoint              | Switch                    | R/W | 1.001, 1.002, 1.003, 1.009 |
-| switch-r       | Digital Readonly DataPoint     | Switch                    | R   | 1.001, 1.002, 1.003, 1.009 |
-| percentage-rw  | Percentage  DataPoint          | Number:Dimensionless      | R/W | 5.001                      |
-| percentage-r   | Percentage Readonly DataPoint  | Number:Dimensionless      | R   | 5.001                      |
-| temperature-rw | Temperature DataPoint          | Number:Temperature        | R/W | 9.001,9.002                |
-| temperature-r  | Temperature Readonly DataPoint | Number:Temperature        | R   | 9.002,9.002                |
-| pressure-r     | Pressure Readonly DataPoint    | Number:Pressure           | R   | 9.006                      |
-| flowrate-r     | Flowrate Readonly DataPoint    | Number:VolumetricFlowRate | R   | 13.002                     |
-| mode-rw        | Mode DataPoint                 | Number:Dimensionless      | R/W | 20.102, 20.103, 20.105     |
-| mode-r         | Mode Readonly DataPoint        | Number:Dimensionless      | R   | 20.102, 20.103, 20.105     |
+> Note:
+Not all available data types of the ISM8 interface are fully supported.
+For the moment, the following data types are implemented:
+
+| Channel type   | Datapoint type                           | Item type                 | R/W | KNX-type's                 |
+|----------------|------------------------------------------|---------------------------|-----|----------------------------|
+| switch-rw      | Digital DataPoint                        | Switch                    | R/W | 1.001, 1.002, 1.003, 1.009 |
+| switch-r       | Digital Readonly DataPoint               | Switch                    | R   | 1.001, 1.002, 1.003, 1.009 |
+| percentage-rw  | Percentage  DataPoint                    | Number:Dimensionless      | R/W | 5.001                      |
+| percentage-r   | Percentage Readonly DataPoint            | Number:Dimensionless      | R   | 5.001                      |
+| value1ucount-r | Value 2-byte Unsigned Readonly DataPoint | Number:Dimensionless      | R   | 5.010                      |
```

Comment:
Can `value1ucount-r` and `value2ucount-2` channel types be combined to a single `nummeric-r` channel type? As both are numbers, just the size is different? Also a little less technicall notation.

There is also a typo:
```suggestion
| value1ucount-r | Value 1-byte Unsigned Readonly DataPoint | Number:Dimensionless      | R   | 5.010                      |
```

Reply:
yes, it is now combined in one number-r channel.
The add-on anyway distinguishes the processing steps depending on the given DPT.